src/test/java/org/folio/qm/conveter/MarcHoldingsQmConverterTest.java

```diff
@@ -59,16 +59,16 @@
   public static final String IND_2 = "ind2";
   private static final Logger logger = LogManager.getLogger(MarcHoldingsQmConverterTest.class);
 
-  @ParameterizedTest
-  @EnumSource(GeneralTestEntities.class)
-  void testRestoreGeneralAndAdditionalCharacteristicsControlFields(GeneralTestEntities testEntity) {
```

Comment:
You write. In the case of different converting of 008 field, this test is not needed.

Reply:
thanks, removed